### DISCUSSION TITLE
2.1 prompts don't appear to work at all.

### DISCUSSION THREAD

**Discussion by rycossette** [TheLastBen/fast-stable-diffusion#1003] [General]
First of all, thank you for your hard work on maintaining this. with 1.5 the training all works perfectly well.

But now I wanted to test out some 2.1, using the `stabilityai/stable-diffusion-2-1` checkpoint.

I was training a model with my face, and one with my wifes face and one with us both together. The images I uploaded were named: rmcosse (1).jpg and kassmar (1).jpg. There are 20 images of myself and 19 of my wife. I just used the default training settings on everything else. Made sure the style checkbox was unchecked. 

I'm testing them locally in Automatic1111, using the 2.0 yaml. 

The results for me look nothing like me, and the results for my wife give me a random man from india. (my wife is as white as they come)

I'm confused about what I'm doing wrong.... Since it appears as though my prompts are just being ignored...

**Comment by TheLastBen** [comment]
resume training and set the text_encoder_steps to 1000 (extreme number), and see if it works, and make sure you are using the latest colab : https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast-DreamBooth.ipynb

and you should know that V2.1 is harder to train, I'm working to find the right learning rate

**Reply by rycossette** [reply]
ok great, thanks for the tip!

I have been able to train it on some retro ray gun images, and that seems to work fine, but people seem to be treated differently. Overall the 2.1 results give better looking faces by default. So something is different there

**Comment by bosimao** [comment]
这是来自QQ邮箱的自动回复邮件。
 
您好，您的信息已传达，祝您天天快乐，日日无忧！
